playbooks/roles/ses-rook/tasks/main.yml

```diff
@@ -103,6 +103,17 @@
   retries: 10
   delay: 45
 
+- name: Check for presence of OSD pods
+  shell: |
+    set -o pipefail
+    kubectl get pods -n rook-ceph -l app=rook-ceph-osd --no-headers=true --field-selector=status.phase=Running | wc -l
+  register: _rook_ceph_osd_pods
+
+- name: Make sure there is some OSD pod running
+  fail:
```

Comment:
do we want to directly fail or could it be that they take a bit to appear?

Also, cna we sync between the checks for existence with the other ones above? I dont mind either way but I would really like to see us using the same approach on all of them :)

Reply:
In all the tests I've done at this point osd pod either already exists (esp. after all those tests passing) or it will not appear.
Anyway: you're suggesting to use similar wait loop like the other tests, right? OK, sure, I can do it.

Reply:
>  Anyway: you're suggesting to use similar wait loop like the other tests, right? OK, sure, I can do it.

Mainly yeah, so its just the same thing all over

Reply:
Updated